ui/src/main/resources/FileManagerCode/Pagination.xml

```diff
@@ -213,6 +220,11 @@
           scope.offset = pageNumber * scope.limit;
         };
 
+        scope.onChangedLimitOption = function() {
+          scope.limit=this.limit;
+          $location.search('filesLimit', scope.limit);
```

Comment:
When updating the query string, the live table is updated but the last page number of the page list is not updated.